Input/GamepadTest/source/PlayState.hx

```diff
@@ -167,12 +188,27 @@ class PlayState extends FlxState
 		}
 	}
 	
+	private function updateTrigger(axis:Int, sprite:FlxSprite, pos:Float):Void
+	{
+		var yAxisValue = _gamePad.getAxis(axis);
+		yAxisValue = (yAxisValue+1) / 2;
+		sprite.y = pos + (10 * yAxisValue);
+	}
+	
 	private function updateDpad():Void
 	{
-		var dpadLeft = _gamePad.pressed(XboxButtonID.DPAD_LEFT);
-		var dpadRight = _gamePad.pressed(XboxButtonID.DPAD_RIGHT);
-		var dpadUp = _gamePad.pressed(XboxButtonID.DPAD_UP);
-		var dpadDown = _gamePad.pressed(XboxButtonID.DPAD_DOWN);
+		#if !flash
```

Comment:
Ah, so you didn't really get the API to behave consistently, just the demo. This is what I meant with this not being a simple button ID update. :)

I really dislike that we can't have a consistent API for this across flash and native (which used to work!), but there isn't much that can be done about it atm I guess...